archean/egg-archean.json

```diff
@@ -0,0 +1,189 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-10-21T19:32:56+00:00",
+    "name": "Archean",
+    "author": "tueem@tomatentum.net",
+    "uuid": "01555bd3-155f-4ab6-b63d-c8bed0a3585f",
+    "description": "Archean invites you to unleash your creativity alone or with friends in a creative sandbox mode.Design and build vehicles, bases, rockets, space stations and much more using customizable modular blocks.Create unique configurations using components, program their behavior and explore an environment free of constraints.\\nTest your builds, refine your designs and defy the laws of physics.Archean is a game in development focused on engineering and build.",
+    "features": [],
+    "docker_images": {
+        "Ubuntu": "ghcr.io\/pelican-eggs\/yolks:ubuntu"
+    },
+    "file_denylist": [],
+    "startup": "echo \"$ADMINS\" > \"Archean-data\/server\/admins.txt\" && cd Archean-server-alpha && .\/Archean server",
```

Comment:
> `cd Archean-server-alpha && ./Archean server`

`./Archean-server-alpha/Archean server` would be cleaner I think, but up to you.

Reply:
Yeah i tried that, but it needs to be started from that directory.

Reply:
Huh, interesting...